src/main/java/edu/rosehulman/covidtracer/model/PositiveCase.java

```diff
@@ -40,11 +41,12 @@
 	public PositiveCase() {
 	}
 
-	public PositiveCase(Person person, Date testDate, Date symptomaticStartDate, Date quarantineEndDate, boolean needsTransportation) {
+	public PositiveCase(Long ID, Person person, Date d, Date d2, Date d3, boolean needsTransportation) {
```

Comment:
Should the names for the dates in the constructor be more descriptive?

Reply:
Oh, oops. That constructor was auto generated and I never changed it. Good catch!

Reply:
Thanks m8! :D